src/app/main/component/events/services/events.service.ts

```diff
@@ -14,6 +14,10 @@ export class EventsService implements OnDestroy {
 
   constructor(private http: HttpClient) {}
 
+  public getImageAsFile(img): Observable<any> {
```

Comment:
you're expecting  `Blob` type in the component, but the type here is specified as `any`